<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Turni_di_lavoro</class>
 <widget class="QWidget" name="Turni_di_lavoro">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>718</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Turni di Lavoro</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	background-color: rgb(254, 235, 156);
}

QCalendarWidget QAbstractItemView:enabled 
{
	border-radius:15px;
	padding: 5px;
	color: rgb(92, 164, 144);  
	background-color:rgb(255, 255, 255); 
	selection-background-color: rgb(92, 164, 144); 
	selection-color:rgb(255, 255, 255); 
}

QCalendarWidget QToolButton {
  	border-radius:14px;
	padding: 5px;
	color: white;
	background-color: rgb(92, 164, 144);
  }

QCalendarWidget QMenu {
  	left: 20px;
  	color: white;
  	background-color: rgb(92, 164, 144);
  }

QCalendarWidget QSpinBox { 
  	color: white; 
  	background-color: rgb(92, 164, 144); 
  	selection-background-color: rgb(55, 92, 108);
  	selection-color: rgb(255, 255, 255);
  }

QCalendarWidget QSpinBox::up-button { 
	background-color: rgb(255, 255, 255);
	border-radius:5px;
	padding: 5px;
	}
  QCalendarWidget QSpinBox::down-button {
	background-color: rgb(255, 255, 255);
	border-radius:5px;
	padding: 5px;
}
  QCalendarWidget QSpinBox::up-arrow { 
	image: url(G:/Secondo anno automazione/Ingegneria del softwear/Cinema/Inerfaccia grafica/ComboBox-up-arrow.png)


}
  QCalendarWidget QSpinBox::down-arrow {
	image: url(G:/Secondo anno automazione/Ingegneria del softwear/Cinema/Inerfaccia grafica/ComboBox-arrow.png)

}
QCalendarWidget QWidget#qt_calendar_navigationbar
{ 
	border-radius:13px;
	padding: 10px;
  background-color:rgb(255, 255, 255); 
}
QCalendarWidget QAbstractItemView:disabled 
{ 
color:rgb(52, 93, 81); 
}

QTableWidget{
	selection-background-color:rgb(123, 220, 193);
	padding: 4px;
}

QHeaderView::section {
    border-radius:10px;
	background-color: rgb(92, 164, 144);
	padding: 4px;
    border: 1px solid #fffff8;
	color: rgb(255, 255, 255);
	font-size: 9pt;
}

QHeaderView::section:hover,QHeaderView::section:checked{
    border-radius:10px;
	background-color:rgb(123, 220, 193);
	padding: 4px;
    border: 1px solid #fffff8;
	color: rgb(255, 255, 255);
	font-size: 9pt;
}

QTableView QTableCornerButton::section {
	 border-radius:10px;
	 border: 2px solid #fffff8;
}





</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QLabel" name="Turni_di_lavoro_2">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(92, 164, 144);</string>
     </property>
     <property name="text">
      <string>Turni di lavoro</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="btn_torna">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>28</width>
         <height>28</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
 border-radius: 14px;
Width:28px;
Height:28px;
background-color: rgb(55, 92, 108);
color: rgb(255, 255, 255);
border-style: outset;


}

QPushButton:hover{

background-color: rgb(92, 154, 181);
}
QPushButton:pressed{
border-style: inset;
background-color: rgb(48, 80, 91);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../../images/Torna-Indietro-icon.png</normaloff>../../../images/Torna-Indietro-icon.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_Home">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>28</width>
         <height>28</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
 border-radius: 14px;
Width:28px;
Height:28px;
background-color: rgb(55, 92, 108);
color: rgb(255, 255, 255);
border-style: outset;


}

QPushButton:hover{

background-color: rgb(92, 154, 181);
}
QPushButton:pressed{
border-style: inset;
background-color: rgb(48, 80, 91);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../../images/Home-icon.png</normaloff>../../../images/Home-icon.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>17</width>
         <height>17</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_12">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>In questa sono elencati i turni di lavoro dei dipendenti</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QTableWidget" name="tabella">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>350</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>500</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="cursor" stdset="0">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;pre style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; line-height:130.769%; font-family:'Courier New';&quot;&gt;&lt;br/&gt;&lt;/pre&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget {
    background-color: rgb(255, 255, 255);
	  border-radius:10px;
}


QTableWidget{
	selection-background-color:rgb(123, 220, 193);
	padding: 4px;
}

QHeaderView::section {
    border-radius:10px;
	background-color: rgb(92, 164, 144);
	padding: 4px;
    border: 1px solid #fffff8;
	color: rgb(255, 255, 255);
	font-size: 9pt;
}

QHeaderView::section:hover,QHeaderView::section:checked{
    border-radius:10px;
	background-color:rgb(123, 220, 193);
	padding: 4px;
    border: 1px solid #fffff8;
	color: rgb(255, 255, 255);
	font-size: 9pt;
}

QTableView QTableCornerButton::section {
	 border-radius:10px;
	 border: 2px solid #fffff8;
}
</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>0</number>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectItems</enum>
       </property>
       <property name="showGrid">
        <bool>true</bool>
       </property>
       <property name="gridStyle">
        <enum>Qt::SolidLine</enum>
       </property>
       <property name="sortingEnabled">
        <bool>false</bool>
       </property>
       <property name="cornerButtonEnabled">
        <bool>true</bool>
       </property>
       <property name="rowCount">
        <number>6</number>
       </property>
       <property name="columnCount">
        <number>1</number>
       </property>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <column>
        <property name="text">
         <string>Turno di lavoro</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QCalendarWidget" name="calendarWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>340</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>300</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
